README.md

```diff
@@ -1 +1,102 @@
-# helllo-microverse
\ No newline at end of file
+# helllo-microverse
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project">Hello Microverse</a>
+
+**Hello Microverse** is a project that was created in relation to the Microverse curriculumn.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:microvese-projects/hello-microverse.git
+```
+-
+
+### Usage <a name="usage"></a>
+
+To run the project:
+
+Open the index.html file in your local browser to interact.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Benson Njuguna**
+
+- GitHub: [@bennyjoez](https://github.com/bennyjoez)
+- Twitter: [@bennyjoezz](https://twitter.com/bennyjoezz)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/microvese-projects/hello-microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse for the outstanding collection of information they have that enabled me to create the project.
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] :+1: You have made a brilliant effort to get this up and running you still need to give your nice README.md some more time and attention so that you can have content that make your project looks professional to visitors or users. Kindly check [this README template](https://github.com/microverseinc/readme-template) again and see all the important sub-titles that you are missing and their respective details.
  - [ ] * The only **optional** section is FAQ.
  * You can delete the live demo section since you are not deploying your application now.
  * Kindly add the section **Future Features**, including 1-3 potential features to be added.
* It will be more customised to **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.